### PR TITLE
[ESLint] - applying default usage of strict

### PR DESCRIPTION
**Pull request by bluong**
This forces javascript files to be in "strict mode".

Invalid:
```javascript
function start() {
//doStuff
}
```

Valid:
```javascript
function start() {
"use strict";
//doStuff
}
```

### CODE REVIEW

File: quicktests/umd/app.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /*jshint strict:false */
+/*eslint strict: 0*/
```
Comment (aicioara) [inline quicktests/umd/app.js]: Can we just `"use strict"` instead of disabling the rule?

Comment (bluong) [inline quicktests/umd/app.js]: Note that there is some message about jshint being set to false...

@jtlan Can you chime in on why this should not be in strict mode?

Comment (jtlan) [inline quicktests/umd/app.js]: I think it's because you have to use "`require`" before it's declared?

File: quicktests/umd/app.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
-/*jshint strict:false */
-
-var require;
+/*eslint-env node */
```
Comment (aicioara) [inline quicktests/umd/app.js]: `/*eslint-env amd*/` ?

Comment (bluong) [inline quicktests/umd/app.js]: Good catch

**Pull request merged**